src/initialization/logs.py

```diff
@@ -0,0 +1,48 @@
+# Log Parser for RTI Connext.
+#
+#   Copyright 2016 Real-Time Innovations, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Create the dictionary for log functions related to network data flow.
+
+Functions:
+  + get_regex_list: Get the regular expressions and function list.
+"""
+from __future__ import absolute_import
+import initialization.initialization as initialization
+import debug.debug as debug
+
+
+def get_regex_list():
+    """Return the regular expressions and functions list for this module."""
+    regex = []
+
+    # NDDS_DISCOVERY_PEERS
+
+    regex.append([initialization.on_no_environment_discovery_peers,
+                  r"RTIOsapi_envVarOrFileGet:environment variable NDDS_DISCOVERY_PEERS not found"])
+    regex.append([initialization.on_no_file_discovery_peers,
+                  r"RTIOsapi_envVarOrFileGet:file NDDS_DISCOVERY_PEERS not found"])
+    regex.append([debug.on_ignored_message,
+                  r"DDS_DomainParticipantFactory_initializeI:Welcome to NDDS"])
+    regex.append([debug.on_ignored_message,
+                  r"DDS_DiscoveryQosPolicy_get_default:no environment variable or file NDDS_DISCOVERY_PEERS"])
+    regex.append([initialization.on_no_ndds_home,
+                  r"RTIOsapi_envVarOrFileGet:environment variable NDDSHOME not found"])
+    regex.append([initialization.on_no_environment_ndds_qos_profiles,
```

Comment:
Check if the `RTIOsapi_envVarOrFileGet` expression are parsed in the config module.